libmodernize/main.py

```diff
@@ -41,11 +44,13 @@ def main(args=None):
     parser.add_option("--no-diffs", action="store_true",
                       help="Don't show diffs of the refactoring.")
     parser.add_option("-l", "--list-fixes", action="store_true",
-                      help="List available transformations.")
+                      help="List standard transformations.")
     parser.add_option("-d", "--doctests_only", action="store_true",
                       help="Fix up doctests only.")
     parser.add_option("-f", "--fix", action="append", default=[],
                       help="Each FIX specifies a transformation; '-f default' includes default fixers.")
+    parser.add_option("-H", "--fixers-here", action="store_true",
```

Comment:
This is totally a matter of taste, but I lean towards having only the long option for this, so people have to spell it out. It's much clearer that way. I quite like the name `--fixers-here`